spec/yaml/swiftnav/sbp/ext_events.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2015 Swift Navigation Inc.
+# Contact: Fergus Noble <fergus@swiftnav.com>
+#
+# This source is subject to the license found in the file 'LICENSE' which must
+# be be distributed together with this source. All other rights reserved.
+#
+# THIS CODE AND INFORMATION IS PROVIDED "AS IS" WITHOUT WARRANTY OF ANY KIND,
+# EITHER EXPRESSED OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND/OR FITNESS FOR A PARTICULAR PURPOSE.
+
+package: swiftnav.sbp.ext_events
+description: |
+  Messages reporting accurately-timestamped external events,
+  e.g. camera shutter time
+
+stable: False
+public: True
+include:
+  - types.yaml
+definitions:
+
+ - MSG_EXT_EVENT:
+    id: 0x0300
+    public: True
+    short_desc: External event
+    desc: |
+        Reports detection of an external event, the GPS time it occurred,
+        which pin it was and whether it was rising or falling.
+    fields:
+        - wn:
+            type: u16
+            units: weeks
+            desc: GPS week number
+        - tow:
+            type: u32
+            units: ms
+            desc: GPS Time of Week rounded to the nearest ms
+        - ns:
+            type: s32
+            units: ns
+            desc: Nanosecond remainder of rounded tow
+        - pin_edge:
```

Comment:
it's the pin and the edge, but sure.. although if there's bitmask syntax that would be better